contracts/safe-hts-precompile/SafeOperations.sol

```diff
@@ -42,8 +42,8 @@ contract SafeOperations {
         SafeHTS.safeTransferNFT(token, sender, receiver, serialNum);
     }
 
-    function safeCryptoTransfer(IHederaTokenService.TokenTransferList[] memory tokenTransfers) external {
-        SafeHTS.safeCryptoTransfer(tokenTransfers);
+    function safeCryptoTransfer(IHederaTokenService.TransferList[] memory transferList, IHederaTokenService.TokenTransferList[] memory tokenTransfers) external {
```

Comment:
Please add test coverage in [SafeHTS tests](https://github.com/hashgraph/hedera-smart-contracts/blob/main/test/safe-hts-precompile/SafeHTS.js)

Reply:
Add a `event CryptoTransfer` that gets emitted after with the response details and you can check for that in the test

Reply:
Actually, I realize now there's a limit to how much we can test at this point without local node also having the services node that supports the functionality.
Had this been a normal contract we wouldn't be tied.
Update the contract but it's okay if we're not able to get a full test right now